File: 2020_2021/L3_S5_2020-2021/INFO0503_modélisation web serv & web avancé/INFO503/TP2_sofiane/Modele.php
<?php
class Modele{

    protected  $modele;

    // constructeur par initialisation

    public function __construct(String $modele){
        $this->modele = $modele;
    }

    // getters

    public function getModele(){
        return $this->modele;
    }

    //setter

    public function set($modele){
        $this->modele = $modele;
    }

    // methode toString

    public function __toString() : String{
        return " nom : " . $this->modele;
    }
}
?>

File: 2020_2021/L3_S5_2020-2021/INFO0503_modélisation web serv & web avancé/INFO503/TP5_Projet_1_01-11-2020/Projet1/src/java/Modele.java
public class Modele{
    private String modele;

    public Modele(String modele){
        this.modele=modele;
    }

    public String getModele(){return modele;}
    
    public void setModele(String modele){this.modele=modele;}

    public String toString(){
        return "nom : " + getModele();
    }
}
